### PR TITLE
Website documentation refinement

### PR DESCRIPTION
**Pull request by craicoverflow**

### CODE REVIEW

File: docs/intro/datamodel.md [hunk 1]
```diff
@@ -3,43 +3,120 @@ id: datamodel
...
-Developers can focus on data and application requirements by modeling them using GraphQL DSL.
+Graphback operates on GraphQL Schema types annotated with `@model`.
+Adding this annotation to your type will enable Graphback to add additional elements to the schema and generate related code in JavaScript or TypeScript.
+
+```graphql
+"""
+@model
+"""
+type Note {
+  ...
```
Comment (wtrocki) [inline docs/intro/datamodel.md]:
```suggestion
  ...
  name: String!
  description: String!
```

File: docs/intro/existingproject.md [hunk 2]
```diff
@@ -11,59 +11,28 @@ Graphback CLI can be used to add code generation capabilities to your existing N
...
-```gql
-type User {
-  id: ID!
-  name: String
-}
-```
+Follow the on-screen instructions.
 
-When ready, run `graphback db` in your terminal to create or update your database structure.
+Graphback will do the following:
```
Comment (wtrocki) [inline docs/intro/existingproject.md]:
```suggestion
`graphback config` will do the following:
```

File: docs/intro/quickstart.md [hunk 3]
```diff
@@ -1,14 +1,37 @@
...
+Or by installing Graphback CLI globally:
+
+```sh
+npm install -g graphback-cli
+graphback init <project-name>
+```
+
+The CLI will ask you to choose from one of two templates:
+
+- `apollo-fullstack-ts`: Apollo GraphQL server and React client using TypeScript.
```
Comment (wtrocki) [inline docs/intro/quickstart.md]: I will add info that we use Postgress.

File: docs/intro/serverless.md [hunk 4]
```diff
@@ -49,7 +49,7 @@ To create GraphQL Layer at runtime developers need to initialize `GraphbackRunti
   });
 ```
 
-See [`./runtime.ts`](https://github.com/aerogear/graphback/blob/master/templates/ts-apollo-runtime-backend/src/runtime.ts) for a fully functional example.
+See our [`ts-apollo-runtime-backend`](https://github.com/aerogear/graphback/blob/master/templates/ts-apollo-runtime-backend/src/runtime.ts) for a fully functional example.
 
 ### GraphbackDataProvider
```
Comment (wtrocki) [inline docs/intro/serverless.md]: This completely outdated. Are we going to do it later?

**Comment by wtrocki** [comment]
Good to merge!

**Comment by wtrocki** [comment]
Let's also delete version and regenerated 0.11 in this branch to reflect the changes

File: docs/intro/serverless.md (hunk 4 above)
Comment (craicoverflow) [reply docs/intro/serverless.md]: Yeah I didn't really get to modify this page.

**Comment by wtrocki** [comment]
Do you need more verification

**Comment by craicoverflow** [comment]
> Do you need more verification

Nope just deleted old website stuff.

**Pull request merged**